Added ANFO and recepies

comma opened the pull request:
Putting BS12 on every watchlist, one PR at a time.

ANFO is an explosive compound made with bascially fertilizer and diesel fuel. So in game that's any fertilizers (EZ,L4Z,Robust) and welder fuel. Different fertilizers give bit more or less output, generally less than components combined.

ANFO is pretty lazy, so to explode it you need /another/ explosion. Explosion power needed depends on how much of ANFO you want to activate.
For volumes under 60u, severity 3 boom will do, 2 for 120u, 1 for more. If you're lucky, a weaker boom can activate part of solution anyway.

Codeside, reagents can now have special interactions when theyir container is being exploded. Moved weldertank's explosion to welderfuel's interaction instead, so I guess technically adding second way to make bombs there.

comma committed:
Added ANFO and recepies

Putting BS12 on every watchlist, one PR at a time.

ANFO is an explosive compound made with bascially fertilizer and diesel fuel. So in game that's any fertilizers (EZ,L4Z,Robust) and welder fuel. Different fertilizers give bit more or less output, generally less than components combined.

ANFO is pretty lazy, so to explode it you need /another/ explosion. Explosion power needed depends on how much of ANFO you want to activate.
For volumes under 60u, severity 3 boom will do, 2 for 120u, 1 for more. If you're lucky, a weaker boom can activate part of solution anyway.

Codeside, reagents can now have special interactions when theyir container is being exploded. Moved weldertank's explosion to welderfuel's interaction instead, so I guess technically adding second way to make bombs there.

Techhead0 commented:
You should also add an ammonia+fuel recipe with 1:1 production. (Actual recipe would include nitric acid too but that's not a thing ingame.)

Anticept commented:
I can see it now:

say ";t how do you make ANFO, I need to blow up the infirmary?"

[Suddenly the MAAs...](https://www.youtube.com/watch?v=LO1Z8HOGeWo)

comma commented:
Added pure ammonia recipie, simplified the checks for non-full activation (fixed them even, they were always failing). It's sufficiently pain in the ass to use for large quantities now, requiring either proper antag explosives or chain of bombs.

NullSnapshot reviewed (approved):
Looks good to me. Glad to see more possible interactions for reagents.
